recording/test/reporter_test.exs

```diff
@@ -166,4 +171,150 @@ defmodule Membrane.RTC.Engine.Endpoint.Recording.ReporterTest do
              }
            } = Reporter.get_report(reporter)
   end
+
+  describe "RTCP synchronization" do
```

Comment:
I would add some comments about how this test should work for future contributors, as it is not obvious how each of these tests works.